examples/mnist_dataset_api.py

```diff
@@ -51,7 +51,7 @@ def cnn_layers(inputs):
 
 batch_size = 128
 buffer_size = 10000
-steps_per_epoch = int(np.ceil(60000 / float(batch_size)))  # = 469
+steps_per_epoch = (60000 + batch_size -1) // batch_size # = 469
```

Comment:
This operation is done once, thus shaving nanoseconds on it is not useful. We should optimize for readability. `np.ceil` is more readable in my opinion.